### DISCUSSION TITLE
How's Dozer doing?

### DISCUSSION THREAD

**Discussion by insanpraja** [getdozer/dozer#2487] [Q&A]
Hi,

The blog section in getdozer.io is partially missing, and docs are not updated. I hope this project still carry on..

Best of luck!

**Comment by reneleonhardt** [comment]
Dozer is an amazing project!

But without more maintainers, contributions are not being merged since 2024, no Dependabot updates, no bugfixes, no new ideas explored like WebAssembly support for Cloudflare Workers and rustls-post-quantum for enhancing security 😞
